cfgov/core/utils.py

```diff
@@ -112,6 +117,13 @@ def add_link_markup(tag):
 
     class_attrs = tag.attrs.setdefault('class', [])
 
+    if request_path is not None:
```

Comment:
Non-blocking, but I think I'd prefer comment above this line with the TODO from the docstring, just so it's in context.

Reply:
Good call. This was harder to do on an earlier iteration where the added code was not a single hunk of stuff, and I didn't think about it after I refactored.

Reply:
Done in 249daf180177650e822290053d4cce0c34824002